.github/workflows/action-test.yaml

```
@@ -644,3 +659,34 @@ jobs:
+      - ipv6
+      - non-web-ports
+      - fail-closed
+      - dns-strict
+      - lockdown
+      - audit
+      - invalid-input
+    if: always()
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check results
+        run: |
```

actionlint (1.7.12)

[error] got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)